f5_openstack_agent/lbaasv2/drivers/bigip/service_adapter.py

```diff
@@ -482,11 +482,6 @@ def _map_virtual(self, loadbalancer, listener,
         self._add_vlan_and_snat(listener, vip)
         self._add_profiles_session_persistence(listener, pool, vip)
 
-        # Set bandwidth controller from loadbalancer's setting.
-        # applying order:
-        #   loadbalancer's bandwidth field -> l7policy's name(bwc name)
-        self._add_listener_bwcPolicy(loadbalancer, vip)
-
         # Set bandwidth controller
         if 'bwcPolicy' in listener:
```

Comment:
Should we also remove these lines?

Reply:
qin, 

485 and 485 were introduced by 304583a3 which we didn't remove for this time.

stack@devstack-agent-lite:~/f5lbaas-agent$ git show 304583a3
commit 304583a3bb6ec65563a1878fdc6c73136498b2e7
Author: Qin Zhao <q.zhao@f5.com>
Date:   Thu Feb 27 16:56:20 2020 +0800

    Support bandwidth control of listener

Reply:
Understand. That is listener level bwc setting. We may re-think it and remove the code in the future.